class-cr-clone.php

```diff
@@ -64,7 +64,17 @@ public function __action_admin_post() {
 			wp_die( esc_html__( 'There was an error copying this post', 'clone-replace' ) );
 		}
 
-		wp_redirect( admin_url( "post.php?post={$post_id}&action=edit" ) );
+		// Modify individual query args for redirect URL.
+		$admin_redirect_url = add_query_arg(
+			apply_filters( 'CR_Clone_redirect_query_args', [
```

Comment:
Filters need phpdoc per [inline documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/) (also applies to line 77).